extensions/notebook/src/prompts/prompt.ts

```diff
@@ -1,10 +1,12 @@
-'use strict';
+/*---------------------------------------------------------------------------------------------
```

Comment:
Are we able to apply these copyrights here? According to the comment below, the code is from somewhere else.

Reply:
I guess I kind of assumed that because we're making changes we should have both. Guess I'll see what CELA has to say - but in the meantime I'll just remove them until we know for sure.